IbisHardwareIGSIO/configio.cpp

```diff
@@ -19,6 +19,10 @@ void ServerConfig::Serialize( Serializer * ser )
 {
     ::Serialize( ser, "ServerName", m_serverName );
     ::Serialize( ser, "IPAddress", m_ipAddress );
+    ::Serialize( ser, "Type", m_type );
+    ::Serialize( ser, "Protocol", m_protocol );
+    ::Serialize( ser, "DeviceName", m_deviceName );
+    ::Serialize( ser, "DeviceType", m_deviceType );
```

Comment:
Why do we need devicename and devicetype. This is getting confusion. Too many things called device.

Reply:
I used the same labels as in OpenIGTLink, but yes it might be confusing to use DeviceName for two different things. In fact DeviceName isn't necessary. We could decide to use the ServerName as DeviceName internally for OpenIGTLink. I didn't want to add restrictions, but it wouldn't be strictly necessary to have both. I can remove it. DeviceType is necessary though. It specifies in OpenIGTLink what kind of messages are being received (e.g. IMAGE, VIDEO). We could again name it differently if we think it's confusing. I just used the same name as was used internally in OIGTL.